_ext/eclipse-cdt/gradle.properties

```diff
@@ -1,21 +1,20 @@
 # Versions correspond to the Eclipse-CDT version used for the fat-JAR.
 # See https://www.eclipse.org/cdt/ for further information about Eclipse-CDT versions.
 # Patch version can be incremented independently for backward compatible patches of this library. 
-ext_version=9.8.0
+ext_version=9.9.0
 ext_artifactId=spotless-eclipse-cdt
-ext_description=Eclipse's CDT C/C++ formatter bundled for Spotless
+ext_description=Eclipse''s CDT C/C++ formatter bundled for Spotless
```

Comment:
not a huge deal, but a typo here, I can fix it but didn't want to mess with your work

Reply:
The syntax highlighter in VSCode seemed to think I needed to escape the single-quote.  That surprised me, and I think it turns out to be wrong.  A fix would be appreciated!

- [.properties spec does not need double-quote](https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html)
- [.properties resource bundle does need double-quote](https://www.mscharhag.com/java/resource-bundle-single-quote-escaping)